shared/libraries/utils.cc

```diff
@@ -59,3 +59,9 @@ bool FloatEdgeDetector::edge() { return posEdge_ || negEdge_; }
 bool FloatEdgeDetector::posEdge() { return posEdge_; }
 
 bool FloatEdgeDetector::negEdge() { return negEdge_; }
+
+int16_t float_to_uint(float x, float x_min, float x_max, int bits) {
```

Comment:
The function name is float_to_uint and "uint" refers to unsigned int, but this function returns signed int. Can we make this consistent?

Reply:
fixed